pkg/subctl/cmd/verify.go

```diff
@@ -76,7 +74,7 @@ func addVerifyFlags(cmd *cobra.Command) {
 }
 
 var verifyCmd = &cobra.Command{
-	Use:   "verify <kubeConfig1> <kubeConfig2>",
+	Use:   "verify <kubeContext1> <kubeContext1>",
```

Comment:
There is already `kubecontext` flag, why not be consistent with it and use it, and in the command add a `kubecontext2` flag (or whatever name makes sense) and make sure they're properly set?

For backwards compatibility you can still accept the kubeconfig args, but those will be optional and only checked if the flags werent specified.

Reply:
Stephen is not going to be around before we do the release.

Let's get this fixed for now. We can handle the nit later.

Reply:
It's ok because the code also supports kubeconfig files for backwards compatibility,.

Reply:
> Stephen is not going to be around before we do the release.
> 
> Let's get this fixed for now. We can handle the nit later.

It is not a nit but rather a UX issue.

As a user I can run `subctl verify --kubecontext cluster3 cluster1 cluster2` which from a "command line flags" perspective is valid, but is rather confusing.

Instead I propose we use `subctl verify --kubecontext cluster3 --othercontext cluster2` (or some such name) to allow a consistent experience.

> It's ok because the code also supports kubeconfig files for backwards compatibility,.

Yes we can still support that regardless, I'm not talking about that.

Reply:
I’ve changed the PR to use `--kubecontexts` as added by @vthapar elsewhere.